aws/request.go

```diff
@@ -33,13 +32,18 @@ const (
 	ErrCodeRequestCanceled = "RequestCanceled"
 )
 
+// NoBody is a http.NoBody reader instructing Go HTTP client to not include
+// and body in the HTTP request.
+var NoBody = http.NoBody
```

Comment:
This type alias is no longer needed. It was a compatability with Go 1.8. The SDK's code can refer to `http.NoBody` instead now.